.github/workflows/publish.yaml

```diff
@@ -23,17 +23,6 @@ jobs:
           pip install build
       - name: Build package
         run: python -m build --sdist --wheel
-      - name: Tag version in GitHub
```

Comment:
We don't want to tag versions in this pipeline if this runs on tagging lest we end up in an infinite cycle.